ui/js/dfv/src/fields/html-field/index.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
-export const PodsDFVHTML = ( props ) => {
+// @todo this may be an incomplete field component
```

Comment:
@zrothauser This is just literally just rendering the HTML, which I realize now we might have to run through `parse` via `html-to-react`?

Reply:
This is really a placeholder and not implemented yet, but shouldn't need to parse it - I think running it through the sanitizer and then setting with `dangerouslySetInnerHTML` should be enough: https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml